cpp/examples/tpch/utils.hpp

```diff
@@ -0,0 +1,208 @@
+#include <iostream>
```

Comment:
Missing copyrights notice in a couple of files.

Reply:
Done